src/Executor/Executor.php

```diff
@@ -40,6 +40,7 @@ public function __construct(string $endpoint)
         $this->headers = [
             'content-type' => 'application/json',
             'authorization' => 'Bearer ' . App::getEnv('_APP_EXECUTOR_SECRET', ''),
+            'x-opr-addressing-method' => 'anycast-efficient'
```

Comment:
We need a better SDK for this at some point.

Reply:
What do you mean? This is the best approach we could come up with.

Reply:
In future we can sdk-generate this class. Adding to backlog